packages/next/src/build/webpack-config.ts

```diff
@@ -1982,6 +1982,7 @@ export default async function getBaseWebpackConfig(
           isNodeServer,
           middlewareMatchers,
           omitNonDeterministic: isCompileMode,
+          clientRouterFilters,
```

Comment:
this is in the build manifest instead now we could clean up the type for this in define plugin now

Reply:
https://github.com/vercel/next.js/blob/canary/packages/next/src/shared/lib/router/router.ts#L1043
https://github.com/vercel/next.js/blob/7678bb904ebc533576394afdace72040dc3cb0be/packages/next/src/build/webpack-config.ts#L2077